src/app/view/perspectives/search/documents/search-documents.component.ts

```diff
@@ -45,28 +46,38 @@ export class SearchDocumentsComponent implements OnInit, OnDestroy {
   @ViewChild('xlTemplate') xlTempl: TemplateRef<any>;
 
   public size: SizeType;
-  public documents: SearchDocument[];
-  private routerSubscription: Subscription;
+  public documents: DocumentModel[];
+  private querySubscription: Subscription;
 
   constructor(private searchService: SearchService,
               private store: Store<AppState>,
               private collectionService: CollectionService,
               private userSettingsService: UserSettingsService) {
+    this.querySubscription = this.store.select(selectQuery)
+      .subscribe(query => {
+        this.store.dispatch(new CollectionsAction.Get({query: query}));
+        this.store.dispatch(new DocumentsAction.Get({query: query}));
+      });
+    Observable.combineLatest(this.store.select(selectAllDocuments), this.store.select(selectAllCollections))
+      .subscribe((values) => {
+        const [documents, collections] = values;
```

Comment:
Cannot we put `[documents, collections]` directly to this array function parameters?